src/Controls/src/Core/Platform/iOS/Extensions/FormattedStringExtensions.cs

```diff
@@ -74,14 +74,12 @@ public static NSAttributedString ToNSAttributedString(
 				return new NSAttributedString(string.Empty);
 
 			var style = new NSMutableParagraphStyle();
-			var lineHeight = span.LineHeight >= 0
+			var lineHeight = span.LineHeight >= 1
 				? span.LineHeight
 				: defaultLineHeight;
 
-			if (lineHeight >= 0)
-			{
-				style.LineHeightMultiple = new nfloat(lineHeight);
-			}
+			// Prevent the span text from being cut off from the top when the specified LineHeight is between 0 and 1.
+			style.LineHeightMultiple = lineHeight >= 1 ? new nfloat(lineHeight) : -1;
```

Comment:
If we change the "-1" to "0" on this line does it all still work?

Reading the docs
https://developer.apple.com/documentation/uikit/nsmutableparagraphstyle/1524596-lineheightmultiple

it seems like anything below zero is not really a valid value. 
"0" is the default so I wonder if we should just set it to "0" here instead of "-1" 

Also, if "0" works then I think we can remove the comment on this method here
https://github.com/dotnet/maui/pull/24273/files#diff-33aadd9d500750cbd0e2664aec8d954c94f8f799d8228d748a196c1070e853fbR62

![image](https://github.com/user-attachments/assets/9f6d720d-6d83-4ede-80c7-ea88b0a29ae8)

Side note :-/ 

it looks line height doesn't work at all on android

![image](https://github.com/user-attachments/assets/cdb43b14-ff5c-4058-88e6-b591e1738c84)

Does it seem like this is also broken on Android for you?